--- origsrc/gconf-cleaner-0.0.3/configure.ac	2007-08-01 19:17:41.000000000 -0500
+++ src/gconf-cleaner-0.0.3/configure.ac	2007-08-02 23:12:05.093750000 -0500
@@ -1,7 +1,7 @@
 dnl Process this file with autoconf to produce a configure script.
 AC_INIT([gconf-cleaner], 0.0.3, [akira@tagoh.org])
 
-. ./requires
+. ${srcdir}/requires
 
 AM_INIT_AUTOMAKE($PACKAGE_NAME, $PACKAGE_VERSION)
 AM_MAINTAINER_MODE
--- origsrc/gconf-cleaner-0.0.3/src/Makefile.am	2007-07-16 22:17:20.000000000 -0500
+++ src/gconf-cleaner-0.0.3/src/Makefile.am	2007-08-02 23:12:05.125000000 -0500
@@ -4,8 +4,7 @@
 	-DG_LOG_DOMAIN="\"GConf-Cleaner\""	\
 	$(GCLEANER_CFLAGS)			\
 	$(NULL)
-LIBS =						\
-	@LDFLAGS@				\
+LDADD =						\
 	$(GCLEANER_LIBS)			\
 	$(NULL)
 
